docs/resource-collection.rst

```diff
@@ -67,8 +71,13 @@ Index creation (Inventory access and index upload)
 
 **Automated index generation**
 
-*This section will be updated once the
-index creation is automated.*
+The GitHub action assumes necessary AWS permissions via the IAM role
+`GitHubActionsRoleResourceIndexer
+<https://us-east-1.console.aws.amazon.com/iamv2/home?region=us-east-1#/roles/details/GitHubActionsRoleResourceIndexer>`__
+which is obtained using OIDC. This role uses permissions from the IAM policy
+`NextstrainResourceIndexer
+<https://us-east-1.console.aws.amazon.com/iamv2/home?region=us-east-1#/policies/details/arn%3Aaws%3Aiam%3A%3A827581582529%3Apolicy%2FNextstrainResourceIndexer>`__
+to list & read the S3 inventories, as well as upload the new index.
```

Comment:
Maybe these are in Terraform on the `james/resources` base branch and I'm not up-to-date there yet, but these sorts of new resources are the kind of thing I think would be great to have in Terraform from their introduction.

Reply:
Added as to-dos in #748